src/identity/reducers/index.ts

```diff
@@ -45,11 +46,12 @@ export default (state = initialState, action: Actions): CurrentIdentity =>
         const {account, org, user} = action.identity
 
         // Store account information from /identity in state.
-        draftState.account.accountCreatedAt = account.accountCreatedAt
         draftState.account.id = account.id
         draftState.account.name = account.name
-        draftState.account.paygCreditStartDate = account.paygCreditStartDate
         draftState.account.type = account.type
+        draftState.account.accountCreatedAt = account.accountCreatedAt
+        draftState.account.isUpgradeable = account.isUpgradeable
+        draftState.account.paygCreditStartDate = account.paygCreditStartDate
```

Comment:
will the app work if some of these properties are present but others are not? Or are all of these necessary? If there is some degraded experience where if we only get `id` and `name`, but nothing else comes back, then I could see this ordering being better. But if everything is either there or it isn't, I think alphabetization is better here, because it doesn't force the reader to think about the ordering.

One of the benefits of an API layer is to make it so we don't have to have a 1:1 mapping of how things work in the OpenAPI spec. We can order things, or rename things so that they make sense with our preferences and styles and domain model.